README.md

```diff
@@ -18,75 +18,143 @@ _For a website to use these components, it should include a single `.js` file an
 
 Required web components
 
+### Item Add
+
+Enables items to be saleable.
+
+```html
+<s2s-add 
+    sku="abc (required)"
+    title="Thing One (required)"
+    over-order="false (optional)"
+    dynamic-price="true (optional)"
+    dynamic-qty="true (optional)"
+    price="123 (optional)"
+    qty="10 (optional)"
+    description="Wonderful new shiny thing (optional)"
+    img="/some/image/link.jpg (optional)"
+    option-size="6 (optional)"
+    />
+```
```

Comment:
There's a lot going on in the spec above for `s2s-add` that needs further clarification. 

For a start, as per the [comment here](https://github.com/stock2shop/cart/pull/5#issuecomment-773916423), this component can be simplified to just
```
<s2s-add sku="abc">
```

Combined with the other components in the comment linked above (minus more advanced components like `s2s-product` and `s2s-product-list`), it's enough to build a functional shopping cart.

The suggestions you make in this commit 70f76009254eb338b114eb98878911b36a1a6b79 can be implement in a subsequent version.

Reply:
Are you reading the README from [correct version](https://github.com/stock2shop/cart/tree/init_brief)

Reply:
Oops yes, I linked to the oldest commit on this branch not the latest one. The description for the [web components](https://github.com/stock2shop/cart/tree/f5a2c7296f469bbe1eb82b62885e94c87b8ea2b2#web-components) is similar to my comment above 
- I would drop the `layout` prop from the initial spec, it could add quite a lot of complexity, leave it for later
- Make the link use the innerText from the custom element as per [this comment](https://github.com/stock2shop/cart/pull/5#issuecomment-774104315)